examples/custom-payload-builder/src/generator.rs

```diff
@@ -5,8 +5,9 @@ use reth::{
     transaction_pool::TransactionPool,
 };
 use reth_basic_payload_builder::{BasicPayloadJobGeneratorConfig, PayloadBuilder, PayloadConfig};
-use reth_node_api::PayloadBuilderAttributes;
-use reth_payload_builder::{error::PayloadBuilderError, PayloadJobGenerator};
+use reth_payload_primitives::{
+    error::PayloadBuilderError, PayloadBuilderAttributes, PayloadJobGenerator,
+};
```

Comment:
all of them should be re-exported form reth-basic-payload-builder, so we don't need the additional dep